rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -147,7 +147,7 @@ void camera_context::save(utils::serial& ar)
 
 	GET_OR_USE_SERIALIZATION_VERSION(ar.is_writing(), cellCamera);
 
-	ar(notify_data_map, start_timestamp, read_mode, is_streaming, is_attached, is_open, info, attr, frame_num);
+	ar(notify_data_map, frame_timestamp_us, read_mode, is_streaming, is_attached, is_open, info, attr, frame_num);
```

Comment:
This is me trolling in the past but this also does savestates loading.

Reply:
that's just a typo. good catch xD
No idea how this happened